src/Symfony/Component/Messenger/Bridge/Kafka/Callback/CallbackProcessorInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Kafka\Callback;
+
+use RdKafka\KafkaConsumer;
+use RdKafka\Message;
+use RdKafka\Producer;
+use RdKafka\TopicPartition;
+
+/**
+ * @see https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/class.rdkafka-conf.html for more information on callback parameters.
```

Comment:
This link is not working.

It would be wise to document the methods rather than linking to a page that may change over time.